parsers/GT.py

```diff
@@ -0,0 +1,137 @@
+# The arrow library is used to handle datetimes
```

Comment:
Not really familiar with that. I actually met this problematic for the first time when looking on how to deal with the accents in `u'Hidroeléctrica',u'Eólico',u'Geotérmica'`. Should we just add them?